pkg/store/fake_store_test.go

```diff
@@ -263,6 +264,193 @@ func TestFakeStoreIngressV1(t *testing.T) {
 	assert.Len(store.ListIngressesV1beta1(), 0)
 }
 
+func TestFakeStoreIngressClassParams(t *testing.T) {
+	assert := assert.New(t)
+	apiGroup := ingressClassParamsAPIGroup
+	unrelatedAPIGroup := "unrelated.com"
+
+	cases := []struct {
+		name               string
+		ingressClassParams []*kongv1alpha1.IngressClassParams
+		ingressClasses     []*networkingv1.IngressClass
+		shouldExist        bool
+	}{
+		{
+			name:               "When there are no ingress classes defined",
```

Comment:
"When there is" is superfluous - keep the descriptions simple: e.g. `no ingress class defined`